runtime/vm/exceptiondescribe.c

```diff
@@ -220,6 +220,27 @@ printStackTraceEntry(J9VMThread * vmThread, void * voidUserData, UDATA bytecodeO
 	return TRUE;
 }
 
+typedef struct RomToRamEntry {
```

Comment:
For the entry stored to the hashtable, we don't need to store both the romClass address and ramClass address. As ramClass has pointer to romClass already. 

Similar to `romClassTableQueryEntry/romClassTableEntry`, `KeyHashTableClassQueryEntry/KeyHashTableClassEntry`, you can have a `RomToRamQueryEntry` and `RomToRamEntry`. `RomToRamQueryEntry` contains only a `J9ROMClass*` and `RomToRamEntry` contains only a `J9Class *`.